<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MoleQueue</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="MoleQueue::ProgramTableView" name="jobView">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
      <attribute name="headerStretchLastSection">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionTest"/>
    <addaction name="actionMove"/>
    <addaction name="actionQueueManager"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="fileMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionTest">
   <property name="text">
    <string>&amp;Test</string>
   </property>
  </action>
  <action name="actionMove">
   <property name="text">
    <string>&amp;Move</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionMinimize">
   <property name="text">
    <string>Mi&amp;nimize</string>
   </property>
  </action>
  <action name="actionMaximize">
   <property name="text">
    <string>Ma&amp;ximize</string>
   </property>
  </action>
  <action name="actionRestore">
   <property name="text">
    <string>&amp;Restore</string>
   </property>
  </action>
  <action name="actionQueueManager">
   <property name="text">
    <string>Queue &amp;Manager</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MoleQueue::ProgramTableView</class>
   <extends>QTreeView</extends>
   <header location="global">MoleQueue/ProgramTableView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
